next/docs/events/pag2025.mdx

```diff
@@ -0,0 +1,31 @@
+---
+conference: "PAG 32"
+description: "The Plant and Animal Genome Conference (PAG) is designed to provide a forum on recent developments and future plans for plant and animal genome projects worldwide."
+eventType: "Conference"
+featured: true
+location: "The Town & Country Resort, 500 Hotel Cir N, San Diego, CA, USA"
+sessions:
+  [
+    { sessionEnd: "10 Jan 2025 9:00 AM", sessionStart: "10 Jan 2025 5:00 PM" },
+    { sessionEnd: "11 Jan 2025 9:00 AM", sessionStart: "11 Jan 2025 5:00 PM" },
+    { sessionEnd: "12 Jan 2025 9:00 AM", sessionStart: "12 Jan 2025 5:00 PM" },
+    { sessionEnd: "13 Jan 2025 9:00 AM", sessionStart: "13 Jan 2025 5:00 PM" },
+    { sessionEnd: "14 Jan 2025 9:00 AM", sessionStart: "14 Jan 2025 5:00 PM" },
+    { sessionEnd: "15 Jan 2025 9:00 AM", sessionStart: "15 Jan 2025 5:00 PM" },
+  ]
+timezone: "America/Los_Angeles"
+title: "Plant and Animal Genome Conference"
+---
+
+<EventsHero {...frontmatter} />
+
+The Plant and Animal Genome Conference (PAG) is designed to provide a forum on recent developments and future plans for plant and animal genome projects worldwide. The conference is attended by scientists from 65 countries worldwide, and is an excellent opportunity to exchange ideas and applications on important research from all over the world.
+
+PAG 32 will be a 6-day conference featuring 7 plenary talks presented by world renown authorities in plant and animal genomics, 200+ scientific workshop sessions, a gallery of 1,000+ poster abstracts, and a vendor expo of 100+ leading companies and organizations in genomics industries. [Register Today](https://pag.confex.com/pag/32/registration/index.cgi) for our best rates.
+
+## Event Details
+
+- Conference website: [https://www.intlpag.org/2025/](https://pag.confex.com/pag/32/registration/index.cgi?username=182044&password=719200)
+- Agenda: [Titles, dates, times, speakers, and length of sessions]
+- How to register: [https://pag.confex.com/pag/32/registration/index.cgi?username=181425&password=618028](https://pag.confex.com/pag/32/registration/index.cgi?username=181425&password=618028)
```

Comment:
@esthermcdade could you please update the registration URL to `https://pag.confex.com/pag/32/registration/index.cgi`.